deploy-manage/deploy/cloud-enterprise/manage-elastic-stack-versions.md

```diff
@@ -35,7 +35,8 @@ $$$ece-elastic-stack-stackpacks-recent$$$
 
 | Required downloads | Minimum required ECE version |
 | --- | --- |
-| [Elasticsearch, Kibana, APM, and Enterprise Search stack pack: 8.17.1](https://download.elastic.co/cloud-enterprise/versions/8.17.1.zip) | ECE 3.0.0<br>(+ Docker 20.10.10+ required for 8.16+) |
+| [Elasticsearch, Kibana, APM, and Enterprise Search stack pack: 9.0.0](https://download.elastic.co/cloud-enterprise/versions/9.0.0.zip) | ECE 4.0.0 |
```

Comment:
```suggestion
| [Elasticsearch, Kibana, and APM stack pack: 9.0.0](https://download.elastic.co/cloud-enterprise/versions/9.0.0.zip) | ECE 4.0.0 |
```

Nit-pick: Removing "Enterprise Search" for 9.0.0, since it isn't available anymore.

Reply:
good catch! missed this mention of enterprise search